clients/spark/build.sbt

```diff
@@ -74,7 +74,9 @@ libraryDependencies ++= Seq(
   "com.lihaoyi" %% "os-lib" % "0.7.8" % "test",
   // Test with an up-to-date fasterxml.
   "com.fasterxml.jackson.module" %% "jackson-module-scala" % "2.14.2" % "test",
-  "com.storm-enroute" %% "scalameter" % "0.19" % "test"
+  "com.storm-enroute" %% "scalameter" % "0.19" % "test",
+  "software.amazon.awssdk" % "s3" % "2.20.109",
+  "software.amazon.awssdk" % "auth" % "2.20.109"
```

Comment:
What does this actually means? Is it actually the new AWS v2 sdK? Does this compile it ((instead of provided)? Im a bit confused

Reply:
Removing, tried testing something, and forgot to revert